arangod/IResearch/IResearchLinkHelper.cpp

```diff
@@ -55,6 +55,39 @@
 
 namespace {
 
+bool isIgnoredHiddenEnterpriseCollection(std::string const& cName) {
```

Comment:
I think we can get away with a StringRef here.
This would allow to avoid copying the collection name.

I don't really think this is measurable performancewsie though.
So I think this comment is more FYI,